federation/pkg/kubefed/init/init.go

```diff
@@ -240,6 +241,20 @@ func initFederation(cmdOut io.Writer, config util.AdminConfig, cmd *cobra.Comman
 		if err != nil {
 			return err
 		}
+	}
+
+	//as the federation control plane is up and responding
```

Comment:
Please use the commenting format convention in the future changes. `//` followed by a space and then the comment usually starting with a capitalized letter.